server/config/environments/development.rb

```diff
@@ -76,4 +76,7 @@
 
   # Use a different logger for distributed setups.
   # config.logger = ActiveSupport::TaggedLogging.new(SyslogLogger.new)
+
+  # TODO: Remove assumption that we are developing on a machine that has which installed! Make cross platform...
+  config.os_gemfile_path = File.expand_path(File.join(File.dirname(`which openstudio`), '../Ruby'))
```

Comment:
This will fail on windows, but I'm assuming that no one is developing on windows (or running the development.rb environment. That was the essence of my TODO comment.

Reply:
I guess i'm more concerned with introduction of requirement that bundle install be run locally for the openstudio install.  what is this getting us?

Reply:
i have approved this pr, but for the record i think that at a minimum a very explicit comment about the bundle install requirement will help us in the future.  or commit with this commented out and the explanation to uncomment if you want to run with a locally installed openstudio bundle.